server/internal/graphql/resolver/meshsync.go

```diff
@@ -65,29 +65,23 @@ func (r *Resolver) resyncCluster(ctx context.Context, provider models.Provider,
 			dbHandler.Lock()
 			defer dbHandler.Unlock()
 
-			r.Log.Info("Dropping Meshery Database")
-			err = dbHandler.Migrator().DropTable(
-				&meshsyncmodel.KeyValue{},
-				&meshsyncmodel.Object{},
-				&meshsyncmodel.ResourceSpec{},
-				&meshsyncmodel.ResourceStatus{},
-				&meshsyncmodel.ResourceObjectMeta{},
-				&models.PerformanceProfile{},
-				&models.MesheryResult{},
-				&models.MesheryPattern{},
-				&models.MesheryFilter{},
-				&models.PatternResource{},
-				&models.MesheryApplication{},
-				&models.UserPreference{},
-				&models.PerformanceTestConfig{},
-				&models.SmiResultWithID{},
-				models.K8sContext{},
-			)
+			tables, err := dbHandler.Migrator().GetTables()
 			if err != nil {
 				r.Log.Error(err)
 				return "", err
```

Comment:
Instead of empty string, can you return some valid status?

Reply:
Also use the meshkit error here as well.

Reply:
Sure.
In the file, we have returned many errors without wrapping them into meshkit errors.
Should I modify them all?

Reply:
Ideally yes. Wrap only the errors who are somewhere being logged by Meshkit logger.

Reply:
Okay, will do that.